prow/plugins/trigger/generic-comment.go

```diff
@@ -110,11 +110,11 @@ func handleGenericComment(c Client, trigger *plugins.Trigger, gc github.GenericC
 		}
 	}
 
-	toTest, err := FilterPresubmits(HonorOkToTest(trigger), c.GitHubClient, gc.Body, pr, c.Config.Presubmits[gc.Repo.FullName])
+	toTest, toSkip, err := FilterPresubmits(HonorOkToTest(trigger), c.GitHubClient, gc.Body, pr, c.Config.Presubmits[gc.Repo.FullName])
 	if err != nil {
 		return err
 	}
-	return RunRequested(c, pr, toTest, gc.GUID)
+	return runAndSkipJobs(c, pr, toTest, toSkip, gc.GUID, trigger.ElideSkippedContexts)
```

Comment:
I don't think we need to pass `trigger.ElideSkippedContexts` to this function. Rather, if `trigger.ElideSkippedContexts` is true we should just set `toSkip = nil`.

Reply:
That makes it harder to test and validate, this way we're more certain the functionality is there for every caller of `runAndSkip`

Reply:
It is unclear to me why removing this would make testing harder or how this adds functionality for other callers of `runAndSkip`, but it doesn't really matter right now so I'll take your word for it.
/shrug